package.json

```
@@ -1,55 +1,12 @@
 {
-  "name": "@netlify/NAME",
+  "name": "@netlify/node-template",
   "private": true,
   "version": "0.1.0",
-  "description": "",
-  "files": [
-    "src"
-  ],
-  "scripts": {
-    "test": "run-s format test:dev",
-    "format": "run-s format:check-fix:*",
-    "format:ci": "run-s format:check:*",
-    "format:check-fix:lint": "run-e format:check:lint format:fix:lint",
-    "format:check:lint": "cross-env-shell eslint $npm_package_scriptsArgs_eslint",
-    "format:fix:lint": "cross-env-shell eslint --fix $npm_package_scriptsArgs_eslint",
-    "format:check-fix:prettier": "run-e format:check:prettier format:fix:prettier",
-    "format:check:prettier": "cross-env-shell prettier --check $npm_package_scriptsArgs_prettier",
-    "format:fix:prettier": "cross-env-shell prettier --write $npm_package_scriptsArgs_prettier",
-    "test:dev": "run-s test:dev:*",
-    "test:ci": "run-s test:ci:*",
-    "test:dev:ava": "ava",
-    "test:ci:ava": "nyc -r lcovonly -r text -r json ava"
-  },
-  "scriptsArgs": {
-    "eslint": "--ignore-path .gitignore --cache --format=codeframe --max-warnings=0 \"{src,scripts,.github}/**/*.{js,md,html}\" \"*.{js,md,html}\" \".*.{js,md,html}\"",
-    "prettier": "--ignore-path .gitignore --loglevel=warn \"{src,scripts,.github}/**/*.{js,md,yml,json,html}\" \"*.{js,yml,json,html}\" \".*.{js,yml,json,html}\" \"!**/package-lock.json\" \"!package-lock.json\""
-  },
-  "ava": {
-    "verbose": true
-  },
-  "husky": {
-    "hooks": {
-      "pre-push": "npm run format"
-    }
-  },
-  "keywords": [],
+  "description": "A cookiecutter template to bootstrap NodeJS projects",
```

```suggestion
  "description": "A repository template to bootstrap Node.js projects for Netlify",
```

Some people (like me) might not be familiar with cookiecutter, so it might be clearer to leave out this implementation detail from the description?
Also, it might be good to specify this is specific to our own needs (not meant outside of Netlify)?